src/main/java/org/launchcode/liftoffrecipetracker/controllers/CategoryController.java

```diff
@@ -0,0 +1,50 @@
+package org.launchcode.liftoffrecipetracker.controllers;
+
+import org.launchcode.liftoffrecipetracker.data.CategoryRepository;
+import org.launchcode.liftoffrecipetracker.models.Category;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.validation.Valid;
+
+@Controller
+@RequestMapping("category")
+public class CategoryController {
+
+@Autowired
+private CategoryRepository categoryRepository;
+
+    @GetMapping
+    public String displayAllCategories(Model model) {
+        model.addAttribute("title", "All Categories");
+        model.addAttribute("category", categoryRepository.findAll());
+        return "category/index";
+    }
+
+    @GetMapping("create")
+    public String renderCreateCategoryForm(Model model) {
```

Comment:
For this method name, I would switch it from renderCreateCategoryForm to displayCreateCategoryForm. They're functionally the same, but you use "display" above and I think we should maintain a standardization throughout the project!

Reply:
done!